src/main/java/org/broadinstitute/hellbender/utils/LoggingUtils.java

```diff
@@ -0,0 +1,80 @@
+package org.broadinstitute.hellbender.utils;
+
+import com.google.api.client.repackaged.com.google.common.annotations.VisibleForTesting;
+import htsjdk.samtools.util.Log;
+import com.google.common.collect.EnumHashBiMap;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.LoggerConfig;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+
+/**
+ * Logging utilities.
+ *
+ * Hellbender tools use the Picard Log.LogLevel enum as the type for VERBOSITY command line arguments (ideally
+ * we would use a log4j level enum, but each log4j level is represented by a static object so there is no
+ * proper built-in enum that is compatible with the command line argument framework). Therefore we use the
+ * Picard enum as the Hellbender currency and convert back and forth between that and the log4j namespace
+ * as necessary. Note that the two namespaces have very similar, but not identical level names, and log4j
+ * has a wider set of level values than those supported by the Picard enum.
+ */
+ public class LoggingUtils {
+
+    // Map between the logging level used throughout Hellbender code (which is the Picard Log.LogLevel enum),
+    // and the log4j log Level values.
+    private static EnumHashBiMap<Log.LogLevel, Level> loggingLevelNamespaceMap;
+    private static String badLevelValue = "Unrecognized verbosity level. Must be one of (DEBUG/ERROR/INFO/WARNING)";
+
+    private static EnumHashBiMap<Log.LogLevel, Level> getLoggingLevelNamespaceMap() {
+        if (null == loggingLevelNamespaceMap) {
+            loggingLevelNamespaceMap = EnumHashBiMap.create(Log.LogLevel.class);
+
+            loggingLevelNamespaceMap.put(Log.LogLevel.DEBUG, Level.DEBUG);
+            loggingLevelNamespaceMap.put(Log.LogLevel.ERROR, Level.ERROR);
+            loggingLevelNamespaceMap.put(Log.LogLevel.WARNING, Level.WARN);
+            loggingLevelNamespaceMap.put(Log.LogLevel.INFO, Level.INFO);
+        }
+        return loggingLevelNamespaceMap;
+    }
+
+    //  We're leaking the log4j Level type through these APIS, but they're only for testing
+    @VisibleForTesting
+    public static Log.LogLevel levelFromLog4jLevel(Level log4jLevel) {
```

Comment:
I'd just move this into the test class itself since it's never used any where else.  Also, is ShouldNeverReachHereException the right exception for this direction, since there are log4j levels that aren't included in our map?